pkgs/development/python-modules/timm/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, expecttest
+, pytest-timeout
+, huggingface-hub
+, pyyaml
+, torch
+, torchvision
+}:
+
+buildPythonPackage rec {
+  pname = "timm";
+  version = "0.6.12";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "rwightman";
+    repo = "pytorch-image-models";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-RNjCcCnNhtr5a+29Bx+k427a03MSooqvnuiDQ8cT8FA=";
+  };
+
+  propagatedBuildInputs = [
+    huggingface-hub
+    pyyaml
+    torch
+    torchvision
+  ];
+
+  checkInputs = [ expecttest pytestCheckHook pytest-timeout ];
+  pytestFlagsArray = [ "tests" ];
+  # takes too long and tries to download models:
+  disabledTestPaths = [ "tests/test_models.py" ];
```

Comment:
The nix sandbox would block network access, so it's not taking too long, just never.